specification/common-types/resource-management/v6/privatelinks.json

```diff
@@ -134,11 +174,11 @@
           "description": "Array of private link resources",
           "items": {
             "$ref": "#/definitions/PrivateLinkResource"
-          }
+          },
+          "x-ms-identifiers": []
         },
         "nextLink": {
           "type": "string",
-          "format": "uri",
```

Comment:
I wonder why change it. Is this whats recommended for global consistency? Isn't changing formats usually considered breaking? And format uri appropriate for a nextLink property?

Reply:
you are totally rigth, updated!